zxcvbn-test/ZxcvbnTestMesserliExtended.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.IO.Compression;
+
+namespace zxcvbn_test
+{
+    [TestClass]
+    public class ZxcvbnTest_MesserliExtended
+    {
+        private const string CategoryUnitTest = "UnitTest";
+
+        [TestMethod]
+        [TestCategory(CategoryUnitTest)]
+        [Priority(1)]
+        public void ZxcvbnEmptyPasswordMetric()
+        {
+            var passwordChecker = new Zxcvbn.Zxcvbn();
+
+            var result = passwordChecker.EvaluatePassword("");
+
+            Assert.AreEqual(0.0, result.Entropy);
+        }
+
+        [TestMethod]
+        [TestCategory(CategoryUnitTest)]
+        [Priority(1)]
+        public void ZxcvbnPasswordMetric()
```

Comment:
Describe the intend of the code.

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/1/commits/7e5beed7364e02b3a814352daba567ce8844b2fb